packages/demo/basic/src/graphql/vote/voteDAO.ts

```diff
@@ -0,0 +1,42 @@
+import baseDAO from '@venncity/base-dao';
```

Comment:
why did we need to add vote?

Reply:
Just to have it more complete with a many to many relation